packages/semantic-ui/src/TextWidget/TextWidget.js

```diff
@@ -1,48 +1,57 @@
 /* eslint-disable react/prop-types */
 import React from "react";
-import PropTypes from "prop-types";
 import { Form } from "semantic-ui-react";
 import { getSemanticProps } from "../util";
-import { utils } from "@rjsf/core";
+import {  utils } from "@rjsf/core";
 const { getDisplayLabel } = utils;
 function TextWidget(props) {
   const {
     id,
     placeholder,
-    required,
-    readonly,
-    disabled,
     name,
     label,
-    schema,
-    uiSchema,
     value,
+    required,
+    readonly,
+    disabled,
     onChange,
     onBlur,
     onFocus,
     autofocus,
     options,
+    schema,
+    uiSchema,
     formContext,
   } = props;
-  const semanticProps = getSemanticProps({ formContext, options });
+  const semanticProps = getSemanticProps(
+    { formContext, options,
+      uiSchema,
+      defaultSchemaProps: {
+        fluid: true,
+        inverted: false,
+    }
```

Comment:
Consider moving `options` to a separate line to be consistent with most of the rest of the code. Also, consider not passing `defaultSchemaProps` since it has the same props set as the default value when it is not provided.